test/compress/properties.js

```diff
@@ -539,3 +539,19 @@ first_256_hex_chars_as_properties: {
         };
     }
 }
+
+native_prototype: {
+    options = {
+        unsafe_prototype: true,
+    }
+    input: {
+        Array.prototype.splice.apply(a, [1, 2, b, c]);
```

Comment:
I'm still confused as to what an array arg does as the first splice() argument.
```
Syntax
array.splice(start)
array.splice(start, deleteCount)
array.splice(start, deleteCount, item1, item2, ...)
```
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/splice